kolibri/core/assets/test/horizontal-nav-bar-with-overflow.spec.js

```diff
@@ -0,0 +1,116 @@
+import HorizontalNavBarWithOverflowMenu from 'kolibri.coreVue.components.HorizontalNavBarWithOverflowMenu';
+import { shallowMount, mount } from '@vue/test-utils';
+
+function makeWrapper({ propsData } = {}) {
+  return mount(HorizontalNavBarWithOverflowMenu, { propsData });
+}
+
+const longerNavigationList = [
+  {
+    condition: true,
```

Comment:
Perhaps, `isVisible` instead of `condition` may help with understanding what's this attribute good for?